_locales/pl/messages.json

```
@@ -0,0 +1,383 @@
+		"message": "Strona SteamDB"
+	},
+	"options_header_extra_data": {
+		"message": "Dodatkowe dane"
+	},
+	"options_header_buttons_and_links": {
+		"message": "Przyciski i odnośniki"
+	},
+	"options_header_inventory": {
+		"message": "Ekwipunek"
+	},
+	"options_header_trade_offers": {
+		"message": "Oferty wymiany"
+	},
+	"options_header_enhancements": {
+		"message": "Rozszerzenia"
+	},
+	"options_website_explanation": {
+		"message": "Wyjaśnienie"
+	},
+	"options_website_explanation1": {
+		"message": "Gdy ta opcja jest włączona, rozszerzenie zażąda niektórych danych użytkownika (lista życzeń, posiadane, śledzone, ignorowane produkty i produkty w koszyku) ze <a href=\"https://store.steampowered.com/dynamicstore/userdata/\" target=\"_blank\">Sklepu Steam</a> i użyje ich do pokazania wyróżnień na stronie."
+	},
+	"options_website_explanation2": {
+		"message": "Aby to działało, musisz być zalogowany w sklepie."
+	},
+	"options_website_explanation3": {
+		"message": "Jest to w całości zawarte w twojej przeglądarce i nigdy nie jest wysyłane do bazy SteamDB."
+	},
+	"options_website_explanation4": {
+		"message": "Aby zapobiec nadmiernym zapytaniom do Sklepu Steam, dane użytkownika są buforowane w lokalnej pamięci przeglądarki."
+	},
+	"options_website_highlight": {
+		"message": "Wyróżnij posiadane i pożądane produkty na steamdb.info"
+	},
+	"options_website_highlight1": {
+		"message": "Lista życień, obserwowanie, i ignorowane produkty w sklepie Steam bezpośrednio z SteamDB"
+	},
+	"options_website_highlight2": {
+		"message": "<a href=\"https://steamdb.info/freepackages/\" target=\"_blank\">Włącz skrypt darmowych pakietów</a>"
+	},
+	"options_website_highlight3": {
+		"message": "Włącz dodatkowe filtrowanie na <a href=\"https://steamdb.info/sales/\" target=\"_blank\">stronie sprzedaży</a> i <a href=\"https://steamdb.info/bundles/\" target=\"_blank\">stronie zestawów</a>"
+	},
+	"options_extra_data": {
+		"message": "Dodatkowe dane pobrane z steamdb.info"
+	},
+	"options_extra_data_players": {
+		"message": "Po włączeniu opcji \"Wyświetlaj bieżących graczy i maksymalne szczyty na stronach aplikacji\" lub \"Wyświetlaj ostatnią aktualizację składów na stronach aplikacji\" rozszerzenie wykona bezpieczne żądanie do steamdb.info, aby uzyskać aktualną liczbę graczy i ostatnią aktualizację składów."
+	},
+	"options_extra_data_prices": {
+		"message": "Gdy opcja \"Wyświetlaj najniższą zarejestrowaną cenę na stronach aplikacji\" jest włączona, rozszerzenie wykona bezpieczne żądanie do steamdb.info z twoją walutą (z tym, co Steam uzna za twoją walutę), aby uzyskać zarejestrowaną cenę dla określonej gry."
+	},
+	"options_steamdb_lowest_price": {
+		"message": "Wyświetlaj najniższą zarejestrowaną cenę na stronach aplikacji"
+	},
+	"options_online_stats": {
+		"message": "Wyświetlaj bieżących graczy i maksymalne szczyty na stronach aplikacji"
+	},
+	"options_steamdb_last_update": {
+		"message": "Wyświetlaj ostatnią aktualizację składów na stronach aplikacji"

```

```suggestion
		"message": "Wyświetlaj ostatnią aktualizację plików na stronach aplikacji"
```

To jest tricky, depoty na Steamie przypominają trochę archiwa zip, w których są pliki. Nikt nie będzie wiedział czym jest "skład", lepiej pójść o krok dalej.